src/Controls/GlobeControls.js

```diff
@@ -759,26 +770,36 @@ class GlobeControls extends THREE.EventDispatcher {
         }
     }
 
-    handleZoom(event) {
+    handleZoom(event, dollyOrigin) {
         this.player.stop();
         CameraUtils.stop(this.view, this.camera);
 
-        this.updateTarget();
-        const delta = -event.delta;
-        this.dolly(delta);
+        var point = dollyOrigin ?? this.view.getPickingPositionFromDepth(event.viewCoords);        // mouse position
+        var range = this.camera.position.distanceTo(cameraTarget.position);
 
-        const previousRange = this.getRange(pickedPosition);
-        this.update();
-        const newRange = this.getRange(pickedPosition);
-        if (Math.abs(newRange - previousRange) / previousRange > 0.001) {
-            this.dispatchEvent({
-                type: CONTROL_EVENTS.RANGE_CHANGED,
-                previous: previousRange,
-                new: newRange,
-            });
+        range *= (event.delta > 0 ?  this.zoomInScale :  this.zoomOutScale);
```

Comment:
could you encapsulate `(event.delta > 0 ? this.zoomInScale : this.zoomOutScale);` to re-use it?

Reply:
You mean in a function ? Like: 
`

    #zoomScale(delta) {
        return delta > 0 ?  this.zoomInScale :  this.zoomOutScale;
    }`

and then : 
 ```suggestion
         range *= this.#zoomScale(event.delta);
```

Reply:
or in a constant